### PR TITLE
Adding tx fees to Aleo Concepts

### PR DESCRIPTION
**Pull request by aharshbe**
Aligned with the new transaction fee requirement across all transaction types and not just Deploys, this new section of Aleo Concepts breaks down transaction fees by type, size, and denomination.

### CODE REVIEW

File: documentation/concepts/03A_transaction_fees.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+id: transactions
+title: Transactions Fees
+sidebar_label: Transaction Fees
+---
+
+# Transactions Fees
+
+A **transaction fee** is a fee that is required in order to process a transaction on the Aleo network. Currently, there are three types of transaction fees that correspond to each transaction type: **transfer**, **deploy**, and **execute**. Transaction sizes are averages using [Big _O_ notation (on the order of)](https://en.wikipedia.org/wiki/Big_O_notation).
```
Comment (raychu86) [inline documentation/concepts/03A_transaction_fees.md:9]: There should just be **deploy** and **execute**, as **transfer** is technically an execution.

Comment (raychu86) [inline documentation/concepts/03A_transaction_fees.md:9]: > Transaction sizes are...
This line placement is a little unclear. Maybe add this line to the bottom Notes section?

Comment (aharshbe) [reply documentation/concepts/03A_transaction_fees.md]: Done, just removed it!

File: documentation/concepts/03A_transaction_fees.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+A **transaction fee** is a fee that is required in order to process a transaction on the Aleo network. Currently, there are three types of transaction fees that correspond to each transaction type: **deploy** and **execute**. Transaction sizes are averages using [Big _O_ notation (on the order of)](https://en.wikipedia.org/wiki/Big_O_notation).
+
+## Aleo Credits Demonination Table
+
+|Size of Transaction|Denomination|Value|Example|
+|:-:|:-:|:-:|:-:|
+|Byte|microcredit|1|Lowest denomination|
+|Kilobyte (KB)|millicredit|1000 microcredits|Most transaction types average between 3 and 10 millicredits|
+|Megabyte (MB)|credit|1000 millicredits|Assuming a 1:1 USD mapping|
```
Comment (raychu86) [inline documentation/concepts/03A_transaction_fees.md:17]: I would refrain from mentioning USD in relation to credits

File: documentation/concepts/03A_transaction_fees.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+|`Input`|1 KB|16 * 32|512 millicredits|
+|`Output`|1 KB| 16 * 32|512 millicredits|
+|`Proof`|1 KB|1|1 millicredit|
+
+|Total|Size|Multiplier|Total cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction -> Transition`|32 MB|1|32 credits|
+
+Notes
+* The optimistic execute transaction fee may increase to prevent spamming; it is currently stable between 3 and 5 millicredits
```
Comment (raychu86) [inline documentation/concepts/03A_transaction_fees.md:53]: Maybe add: "it is currently stable between 3 and 5 millicredits for a basic `credits.aleo` transfer`

**Comment by raychu86** [comment]
Left some comments. Otherwise, it's looking good 👍

**Comment by aharshbe** [comment]
@raychu86, thank you for the comments! I've made all the changes. Good to merge?

File: documentation/concepts/03A_transaction_fees.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+|`Transition wrapper`|0 KB|1|0 millicredits|
+|`Input`|1 KB|2|2 millicredits|
+|`Output`|1 KB|2|2 millicredits|
+|`Proof`|1 KB|1|1 millicredit|
+
+|Total|Size|Multiplier|Total cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction -> Transition`|5 KB|1|5 millicredits|
+
+### Structure (pessimistic)
```
Comment (raychu86) [inline documentation/concepts/03A_transaction_fees.md:40]: "pessimistic" and "optimistic" might not be the right word. 

What about something like - `minimum and maximum` or `basic and packed` or some combination.

Comment (aharshbe) [reply documentation/concepts/03A_transaction_fees.md:61]: Agreed, I struggled with this myself. Let's go with min and max.

Comment (aharshbe) [reply documentation/concepts/03A_transaction_fees.md]: Changed to "minimum cost example" and "maximum cost example."

**Review by raychu86 — APPROVED** [review]

**Review by collinc97 — APPROVED** [review]

**Pull request merged**